src/mode.js

```diff
@@ -51,18 +51,19 @@ export function setModeForTesting(m) {
  * @return {!ModeDef}
  */
 function getMode_() {
-  const isLocalDev = (location.hostname == 'localhost' ||
-      (location.ancestorOrigins && location.ancestorOrigins[0] &&
-          location.ancestorOrigins[0].indexOf('http://localhost:') == 0)) &&
+  const hostname = location.hostname;
```

Comment:
Sure, Issue #2008 Filed to request this feature. https://github.com/ampproject/amphtml/issues/2008